astropy/visualization/tests/test_norm.py

```diff
@@ -5,7 +5,7 @@
 from numpy import ma
 from numpy.testing import assert_allclose, assert_array_equal, assert_equal
 
-from astropy.utils.compat.optional_deps import HAS_MATPLOTLIB, HAS_PLT
```

Comment:
Oh interesting. Didn't know we already have mixed usage of `HAS_MATPLOTLIB` and `HAS_PLT`. 😬 

Since this is just test code, this diff is probably okay. At least would make grep-ing easier with consistent HAS checks.

Reply:
Sorry, I forgot to tell you !
If you agree that one `HAS_*` constant is better than two, let me know which one you prefer, and I'll happily put the other one to sleep in a follow up PR.